benchmarks/bench_sdbaseline.py

```diff
@@ -0,0 +1,643 @@
+        self.tid = '304a'
+        infile = self.infile
+        datacolumn='float_data'
+        spw=''
+        blmode='fit'
+        blformat='table'
+        blfunc='variable'
+        dosubtract=True
+        with table_manager(infile) as tb:
+            nrow_data = tb.nrows()
+
+        blparam = self.outroot+'.blparam'
+
+        tb.open(tablename=infile, nomodify=False)
+        r2msk = tb.getcell('FLAG', 2)
+        for ipol in [0,1]:
+            for ichan in range(len(r2msk[0])):
+                r2msk[ipol][ichan] = True
+        tb.putcell('FLAG', 2, r2msk)
+        tb.close()
+        pol = ''
+
+        outfile = self.outroot+self.tid+blfunc+'.ms'
+        bloutput= self.outroot+self.tid+blfunc+'.bltable'
+        sdbaseline(infile=infile,datacolumn=datacolumn,
+                            blmode=blmode,blformat=blformat,bloutput=bloutput,
+                            spw=spw,pol=pol,blfunc=blfunc,blparam=blparam,
+                            dosubtract=dosubtract,outfile=outfile)
+                                 
+    def time_fit_variable_masked_unselect(self):
+        """sdbaseline: testing shortening baseline table for blfunc=variable - test304"""
+        self.tid = '304b'
+        infile = self.infile
+        datacolumn='float_data'
+        spw=''
+        blmode='fit'
+        blformat='table'
+        blfunc='variable'
+        dosubtract=True
+        with table_manager(infile) as tb:
+            nrow_data = tb.nrows()
+    
+        blparam = self.outroot+'.blparam'
+
+        tb.open(tablename=infile, nomodify=False)
+        r2msk = tb.getcell('FLAG', 2)
+        for ipol in [0]:
+            for ichan in range(len(r2msk[0])):
+                r2msk[ipol][ichan] = True
+        tb.putcell('FLAG', 2, r2msk)
+        tb.close()
+        pol = 'RR'
+
+        outfile = self.outroot+self.tid+blfunc+'.ms'
+        bloutput= self.outroot+self.tid+blfunc+'.bltable'
+        sdbaseline(infile=infile,datacolumn=datacolumn,
+                            blmode=blmode,blformat=blformat,bloutput=bloutput,
+                            spw=spw,pol=pol,blfunc=blfunc,blparam=blparam,
+                            dosubtract=dosubtract,outfile=outfile)
+        
+    def time_fit_variable_unselect_masked(self):
+        """sdbaseline: testing shortening baseline table for blfunc=variable - test304"""
+        self.tid = '304c'
+        infile = self.infile
+        datacolumn='float_data'
+        spw=''
+        blmode='fit'
+        blformat='table'
+        blfunc='variable'
+        dosubtract=True
+        with table_manager(infile) as tb:
+            nrow_data = tb.nrows()
+    
+        blparam = self.outroot+'.blparam'
+
+        tb.open(tablename=infile, nomodify=False)
+        r2msk = tb.getcell('FLAG', 2)
+        for ipol in [1]:
+            for ichan in range(len(r2msk[0])):
+                r2msk[ipol][ichan] = True
+        tb.putcell('FLAG', 2, r2msk)
+        tb.close()
+        pol = 'LL'
```

Comment:
Another place that seems like it could be part of a setup function so that measurement is isolated to only the relevant `sdbaseline` call.